example/example2.c

```diff
@@ -59,7 +59,11 @@ int main()
 	nsvgRasterize(rast, image, 0,0,1, img, w, h, w*4);
 
 	printf("writing svg.png\n");
- 	stbi_write_png("svg.png", w, h, 4, img, w*4);
+	stbi_write_png("svg.png", w, h, 4, img, w*4);
+	printf("writing svg.bmp\n");
```

Comment:
I don't think these extra formats are necessary. If you need it for your own app, please put them there instead.